Libraries/LibGUI/ProcessChooser.cpp

```diff
@@ -91,10 +92,31 @@ ProcessChooser::ProcessChooser(const StringView& window_title, const StringView&
     };
 
     table_view.model()->update();
+
+    if (m_refresh_enabled) {
```

Comment:
We are in the constructor here, so will I have no chance to change `m_refresh_enabled` before this happens.
So in practice, there's no way to turn off auto-refresh.

Reply:
I have thought of this but I'm not sure when to start a timer? Should I move this logic to another method named `populate_data` or `fill_table` would be better? So, whoever uses this must call that method to see the data in the table. Does that makes sense?

Reply:
I would suggest just removing the setting and always auto-refreshing the table.

Reply:
Got it !!